### ISSUE TITLE
TfIdfAugmenter does not return the desired number of outputs

### ISSUE THREAD

**Issue by Bunoviske** [makcedward/nlpaug#148]
Hey! I want to share that I am having problems when I need to augment the same sentence more than once (I use the parameter n in the augment function). For some reason, in some samples of my dataset, it does not return 3 outputs as expected, but rather one output (please see the code below). What might be happening? The word_tokenizer I use is the same you provided in the TfIdfTrainer example.

Thanks!

```

tfidfSubstituteAugmenter = naw.TfIdfAug(model_path='.', tokenizer=word_tokenizer, action="substitute", aug_max=None, aug_p=0.3)

sentencesTfidfAug = []

for text in tqdm(texts):
    textSentences = sent_tokenize(text) #split text in sentences
    tfidfResult = tfidfSubstituteAugmenter.augments(textSentences, n=3, num_thread=1)
    sentencesTfidfAug.append(tfidfResult)

    if (len(tfidfResult) != 3):
      print("tfidfResult",len(tfidfResult), tfidfResult)
    

```

**Comment by makcedward** [comment]
For the number of augments function output, it is the size of inputs * n.
For example, if "textSentences" includes 2 sentences and n is 3, the size of "tfidfResult" is 6 (2*3)

**Comment by makcedward** [comment]
Fixed in 1.0.0 version
